themes/src/main/resources/theme/base/admin/resources/partials/role-list.html

```diff
@@ -6,20 +6,20 @@ <h1>Roles</h1>
         <li><a href="#/realms/{{realm.realm}}/default-roles">{{:: 'default-roles' | translate}}</a></li>
     </ul>
 
-    <table class="datatable table table-striped table-bordered dataTable no-footer">
+    <table class="table table-striped table-bordered">
         <thead>
             <tr>
                 <th class="kc-table-actions" colspan="5">
                     <div class="form-inline">
                         <div class="form-group">
                             <div class="input-group">
-                                <input type="text" placeholder="{{:: 'search.placeholder' | translate}}" data-ng-model="searchQuery" class="form-control search" onkeyup="if (event.keyCode === 13){$(this).next('I').click(); }">
+                                <input type="text" placeholder="{{:: 'search.placeholder' | translate}}" data-ng-model="query.search" ng-model-options="{debounce: 500}" class="form-control search">
```

Comment:
Is there any reason for using `ng-model-options="{debounce: 500}"`? AFAIK we're not using it but here.

Reply:
It's related to the automatic search idea, automatic search is perform onlyu if more than 3 character are pressed ( to avoid useless request ) and the debounce is a delay of 500 ms, to avoid unwanted search as the user is typing. But if I rollback the automatic search I will remove the debounce too because it become useless without the automatic search